src/app/helptext/storage/volumes/datasets/dataset-acl.ts

```diff
@@ -55,12 +55,24 @@ dataset_acl_uid_tooltip: T('User who controls the dataset. This user\
  attributes. Users created manually or imported from a directory service\
  appear in the drop-down menu.'),
 
+ apply_user: {
+  placeholder: T('Apply User'),
+  tooltip: T('Set to confirm changes to <i>User</i>. To prevent errors, changes to <i>User</i> \
```

Comment:
```suggestion
  tooltip: T('Confirm changes to <i>User</i>. To prevent errors, changes to the <i>User</i> \
```

Reply:
I kinda think that 'only' is in the wrong place. It's not that we'll only submit and do no other thing, but that we will submit, but only IF...

Reply:
Makes sense to me, no problem!